import-scripts/update-az-mskimpact.sh

```diff
@@ -246,6 +246,17 @@ function standardize_clinical_data() {
     mv "$SAMPLE_OUTPUT_FILEPATH" "$SAMPLE_INPUT_FILEPATH"
 }
 
+function standardize_cna_data() {
+    DATA_CNA_INPUT_FILEPATH="$AZ_MSK_IMPACT_DATA_HOME/data_CNA.txt"
+    DATA_CNA_OUTPUT_FILEPATH="$AZ_MSK_IMPACT_DATA_HOME/data_CNA.txt.standardized"
+
+    # Standardize the CNA file to use NA for blank values
+    $PYTHON_BINARY $PORTAL_HOME/scripts/standardize_cna_data.py -f "$DATA_CNA_INPUT_FILEPATH" > "$DATA_CNA_OUTPUT_FILEPATH" &&
```

Comment:
I'm not certain what the trailing '&&" does ... if this is to connect with the lower "mv" command then I would not have the blank lines between and would double indent the lower line